VERSION

```diff
@@ -1 +1 @@
-0.5.0
+0.5.1
```

Comment:
should bump to 0.6.0, since API is changing

Reply:
will do

Reply:
done